AsyncDisplayKit/ASControlNode.mm

```diff
@@ -253,10 +254,6 @@ - (void)addTarget:(id)target action:(SEL)action forControlEvents:(ASControlNodeE
   NSParameterAssert(controlEventMask != 0);
   
   ASDN::MutexLocker l(_controlLock);
-  
```

Comment:
We need to update the documentation for `_controlEventDispatchTable`. I think something like:

```objectivec
// ASControlEvent -> [TargetAction0, TargetAction1, ... ]
NSMutableDictionary<id<NSCopying>, NSMutableArray<ASControlTargetAction *> *> *_controlEventDispatchTable;
```

is plenty.